frontend/routes/components/fontAwesomeIcon/fontAwesomeIcon.styles.ts

```diff
@@ -43,4 +43,12 @@ export const IconContainer = styled(Icon).attrs({
 	&.primary {
 		color: ${MuiTheme.palette.primary.main}
 	}
+
+	&.fontSizeSmall {
```

Comment:
@dleszcz Why didn't you add `className` prop to this component? I think you can use `styled-components` here.